galaxy_ng/app/api/ui/viewsets/execution_environment.py

```diff
@@ -80,6 +80,7 @@ def get_queryset(self):
         base_path = self.kwargs["base_path"]
         repo = get_object_or_404(models.ContainerDistribution, base_path=base_path).repository
         repo_version = repo.latest_version()
+        # TODO: Fix this with get_content in pulpcore 3.11
```

Comment:
Resolving this TODO as now we have 3.11 released? or it will come in a different PR?

Reply:
Done